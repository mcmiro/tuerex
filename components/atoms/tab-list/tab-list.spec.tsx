import * as React from "react";
import {screen, render} from "@testing-library/react";
import TabList from "./";

describe("Test TabList component", () => {
	it("Should pass when children on TimeItem is defined", () => {
		render(<TabList data-testid="tab-list-test">Test</TabList>);

		const tabList = screen.queryByTestId("tab-list-test");
		expect(tabList).toBeDefined();
		expect(tabList).toMatchSnapshot();
	});
});
