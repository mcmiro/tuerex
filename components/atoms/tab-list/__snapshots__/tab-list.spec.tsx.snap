// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test TabList component Should pass when children on TimeItem is defined 1`] = `null`;
